GVFS/GVFS.Common/Git/CredentialManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿using GVFS.Common.Tracing;
+
+namespace GVFS.Common.Git
+{
+    public class CredentialManager : ICredentialManager

```

Comment:
Naming suggestion: The name `CredentialManager` made me think this class interacted with GCM, when it looks like a more accurate description is that it's a collection of methods to help interact with git's credential CLI.

Would a name like `GitCredential`, `GitCredentialVerb`, or `GitCredentialCLI` be a closer match to the purpose/function of this class?  

You're more familiar with this area so feel free to take/leave this suggestion.

Reply:
Yeah - I was worried about that. @kewillford suggested `CredentialStore`, and I liked that.